packages/katello/katello/hostname-change.rb

```diff
@@ -438,6 +442,8 @@ def run
       unless @foreman_proxy_content
         STDOUT.puts "\nUpdating default #{@proxy}"
         proxy_id = self.get_default_proxy_id
+        self.curl_cmd('DELETE', "/api/v2/instance_hosts/#{@old_hostname}")
+        self.curl_cmd('DELETE', "/api/smart_proxies/#{proxy_id}/hosts/#{@old_hostname}")
```

Comment:
Shouldn't we be explicit about api version? Or at least consistent?

Reply:
probably, I think the one I copied from you from IRC, the other from the API docs.